python/cudf/cudf/core/frame.py

```diff
@@ -6998,12 +6998,12 @@ def _drop_rows_by_labels(
             raise KeyError("One or more values not found in axis")
 
         key_df = cudf.DataFrame(index=labels)
-        if isinstance(obj, cudf.Series):
+        if isinstance(obj, cudf.DataFrame):
+            return obj.join(key_df, how="leftanti")
+        else:
```

Comment:
Why did this order flip? It seems out of scope for this PR. Is it to make this work for Index objects too?

Reply:
Yeah somehow adding annotations for this method triggered a `mypy` error for this piece of code.